app/views/assessments/_gradesheet.html.erb

```diff
@@ -1,19 +1,24 @@
-<%= external_javascript_include_tag "jquery", "1.7.2" %>
-<%= external_javascript_include_tag "jquery-ui", "1.8.20" %>
-<%= external_javascript_include_tag "jquery.dataTables", "1.9.2" %>
+<% content_for :stylesheets do %>
+  <%= stylesheet_link_tag "gradesheet1" %>
+<% end %>
+
+<% content_for :javascripts do %>
+  <%= external_javascript_include_tag "jquery", "1.7.2" %>
+  <%= external_javascript_include_tag "jquery-ui", "1.8.20" %>
+  <%= external_javascript_include_tag "jquery.dataTables", "1.9.2" %>
 
-<%= javascript_include_tag "jquery.jeditable" %>
-<%= javascript_include_tag "KeyTable" %>
+  <%= javascript_include_tag "jquery.jeditable" %>
+  <%= javascript_include_tag "KeyTable" %>
 
-<script type="text/javascript">
-  andrewID_col = 1
-  problem_count = <%= @assessment.problems.length %>
-  first_problem_column = 5;
-  first_non_searchable_column = 3 
-</script>
+  <script type="text/javascript">
+    andrewID_col = 1
```

Comment:
Fix what, the gradesheet? This fixes it.  The scripts trusted that jquery had been loaded, but since they weren't in the content_for, jquery was loaded after the application scripts ran.